app/views/messages/_right_column.html.erb

```diff
@@ -0,0 +1,187 @@
+<turbo-frame  id="conversation"
```

Comment:
should this be `dom_id(@conversation)`?

Reply:
No, this is right. The sidebar links frame target="conversation".  Conceptually, the "conversation" frame is the title + messages + composer, all of which are tied to the conversation in some way. It's basically the full right-hand side.